cache_tmp.md

```diff
@@ -0,0 +1,68 @@
+### cache
+
+#### Description
+
+Semaphore 2.0 offers the `cache` utility for storing the dependencies of
+projects. As a result, the `cache` utility can be used for reducing
+installation time by restoring installed packages quickly in order to be
+reused between jobs.
+
+#### Commands and Command Line Parameters
```

Comment:
I would keep things more focused and more easily scannable here. I practice lets split this into two sections:

## Storing dependencies to cache

## Restoring dependencies from cache